.rat-excludes

```diff
@@ -35,6 +35,7 @@ dagre-d3.min.js
 graphlib-dot.min.js
 sorttable.js
 vis.min.js
+vis.js
```

Comment:
But I think we don't need to include "vis.js" in Spark since it's just for debuging. The developers can add them by themselves locally.